atlasdb-jepsen-tests/src/test/java/com/palantir/atlasdb/jepsen/events/EventTest.java

```diff
@@ -68,7 +69,7 @@ public void canDeserialiseInfoReadWithValue() {
         keywordMap.put(Keyword.intern("f"), Keyword.intern(JepsenConstants.START_FUNCTION));
         keywordMap.put(Keyword.intern("process"), Keyword.intern(JepsenConstants.NEMESIS_PROCESS));
         keywordMap.put(Keyword.intern("time"), SOME_TIME);
-        keywordMap.put(Keyword.intern("value"), Keyword.intern(String.valueOf(SOME_VALUE)));
+        keywordMap.put(Keyword.intern("value"), Keyword.intern(String.valueOf(SOME_INTEGER_AS_STRING)));
```

Comment:
We don't need the `String.valueOf()` any more, since `SOME_INTEGER_AS_STRING` is already a string :)